### ISSUE TITLE
auto-sklearn holds requirements.txt indexed from external file

### ISSUE THREAD

**Issue by linjing-lab** [automl/auto-sklearn#1698]
# lack of understanding in this approach
I do develop Python software, such as optimtool hosted on my repository, and if I don't need to clone a repository where the released software was used as a independent module for optimization and machine learning domain, then why stored *requirements.txt* which indexed from external file with the same name.

If users download auto-sklearn as a independent Python software to build pipelines, not to clone the whole repository from main, so how to verify the dependencies with [*requirements.txt*](https://github.com/automl/auto-sklearn/blob/673211252ca508b6f5bb92cf5fa87c6455bbad99/autosklearn/requirements.txt#L1) which wrote `../requirements.txt`, and this str do not automatically import the strs from external file, just seek the dependencies if the external file exists.

# System Details (if relevant)
* The latest version in the main repository or already released.

**Comment by linjing-lab** [comment]
I found AMD chip natively support DirectML powered by DirectX, and if this auto-sklearn will always use CPU as backend and process with numpy, dask, pandas. The true power efficiency can be evaluated by the nerual hardware-accelerated performance or array structs with storage, method reuse, context progamming. I recommand auto-sklearn to lift power and performance from which hardware really perception, reduce the experiments of supervised learning with medium-scale datasets and method parameters, the reality perspectives only reflect in interaction with pretrained model tensors, some trials for accuracy and meta supervised data will make sense on paper with formula boosted trait, not the boosting machine for neural and power.
